app/src/main/java/com/duckduckgo/app/browser/DuckDuckGoRequestRewriter.kt

```diff
@@ -27,7 +28,7 @@ import timber.log.Timber
 interface RequestRewriter {
     fun shouldRewriteRequest(uri: Uri): Boolean
     fun rewriteRequestWithCustomQueryParams(request: Uri): Uri
-    fun addCustomQueryParams(builder: Uri.Builder)
+    fun addCustomQueryParams(currentUrl: String, builder: Uri.Builder)
```

Comment:
because we need to extract parameters from the current url we need to pass this as a collaborator